pkg/esp32_sdk/patches/0028-components-fix-format-specifier.patch

```diff
@@ -0,0 +1,66 @@
+From 5b62cf49d7ef21474b2b467801ddb95df7a5abcb Mon Sep 17 00:00:00 2001
+From: Marian Buschsieweke <marian.buschsieweke@ovgu.de>
+Date: Mon, 10 Oct 2022 13:09:07 +0200
+Subject: [PATCH 1/4] components: fix format specifier
+
+In various places `"%u"` or `"%x"` are used instead of `"%" PRIu32` and
```

Comment:
Is this really an issue or just a cleanup?

Reply:
It is an issue: `uint32_t` can also be `unsinged long` on 32 bit systems - and apparently is here. This causes GCC to complain about mismatching type and format specifier.

Reply:
O, I see. In former GCC version using `%u` and `%x` was working since `uint32_t` is `unsigned int` on Xtensa-based ESP32x SoCs.